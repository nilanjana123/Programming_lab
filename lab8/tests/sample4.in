#tmmx
# This line is a comment
# This file has a scalar (123), and a 2 x 3 matrix (elements 1 ... 6)

2 3
1,2,3,
4,5,6,

5

2 3
1,2,3,
4,5,6,

