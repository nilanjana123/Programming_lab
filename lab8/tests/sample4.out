# cmd options :tmmx
# After transpose.
3 2
1,4,
2,5,
3,6,

# After multiplication.
3 2
5,20,
10,25,
15,30,

# After multiplication.
3 3
85,110,135,
110,145,180,
135,180,225,

# After determinant.
0
